### ISSUE TITLE
What to run to reproduce the best results?

### ISSUE THREAD

**Issue by bryant1410** [m-bain/frozen-in-time#12]
Given that you do curriculum learning, I was wondering if you could provide the series of config files you use to train the best model in your paper. My understanding is that you first use the CC3M+WebVid config in this repo, which uses 1 frame per video. But then I think you fine-tune on 4 frames, then on 8, right? Is it the same config (iterations, etc.) but changing the frame count and batch size?

**Comment by m-bain** [comment]
Good question, so generally we found curriculum learning to be useful for reducing computation, but typically achieves approximately similar performance to training without curriculum. Our best model was trained on 1-frame then finetuned on 4-frames.
The procedure from this was fairly heuristic: train on 1-frame until the training loss converges, then finetune on 4-frames with the same config like you said but changing frame count and batch size (use load checkpoint). The finetuning requires far less iterations because most of the knowledge is learned in 1-frame setting.

**Comment by bryant1410** [comment]
Thanks for the clarification! Maybe you could add this info in the README?
